third_party/docfx/templates/devsite/partials/class.tmpl.partial

```diff
@@ -2,7 +2,6 @@
 
 {{>partials/class.header}}
 {{#children}}
-<h2 id="{{id}}">{{>partials/classSubtitle}}</h2>

```

Comment:
Not following this change -- why is this deleted?

(Not saying it's wrong to delete necessarily, but I don't understand it yet.)

Reply:
More thoughts: this avoids having two headers with no content in between. However, these headings provide relevant semantic meaning for the page: functions, methods, constants, variables, types, and so on. It seems confusing if we drop those headers and have the underlying elements all at the same level.

To make it explicit, this `Constants` header is being deleted, which makes the page more difficult to understand: https://github.com/googleapis/doc-templates/pull/160/files#diff-3f1b5a3a83b83d7e838c9c0763f5caa5eb1cb7fe0bc6bb624c004ae2a3ea4e5dL251

cc @jennyrosewood, please take a look.

Reply:
The headers were removed as suggested, and I can see this going either way:

Removing the headers does seem to make it quite hard to differentiate between sections. However it does help include some of the sub-headers (for example, for Go) to show up in the Table of Contents as we bumped some of them from h4 to h3. Removing the h2 headers required all the `partials/[namespace|class]Subtitle` lines to be removed.

Keeping the headers would help differentiate the sections. However that would cause some sub-headers to no longer appear in the Table of Contents as they would be pushed to h4.

Reply:
I would prefer to have the differentiated sections over having the sub-headers show up in the TOC. The separation is key to understanding the content on the page.

I would be OK with getting rid of the `Types` header and bumping the elements within it up one level. We'd still keep the `Variables` and `Constants` and other headers, though. That way, you would see a type and its methods/functions/etc. in the TOC.

Reply:
Sounds good. Moving forward with just Types gone.